docs/content/preview/api/ysql/the-sql-language/statements/dml_declare.md

```diff
@@ -50,7 +50,7 @@ See the generic section [Cursors](../../../cursors/).
 
 ## Semantics
 
-`DECLARE` creates a _cursor_.  (There's no notion of so-called "opening" a _cursor_ in top-level SQL.) A _cursor's_ duration is limited to the duration of the session that declares it. (The [`CLOSE`](../dml_close) statement drops a _cursor_ so that you can shorten its lifetime if you want to—typically in order to save resources.) Notice the critical maximum lifetime difference between a _holdable_, and a _non-holdable_, _cursor_.
+`DECLARE` creates a _cursor_.  (There's no notion of so-called "opening" a _cursor_ in top-level SQL.) A _cursor's_ duration is limited to the duration of the session that declares it. Notice the critical maximum lifetime difference between a _holdable_, and a _non-holdable_, _cursor_. See the section [Transactional behavior — holdable and non-holdable cursors](../../../cursors/#transactional-behavior-holdable-and-non-holdable-cursors). (The [`CLOSE`](../dml_close) statement drops a _cursor_ so that you can shorten its lifetime if you want to—typically in order to save resources.)
 
 The _pg_cursors_ catalog view lists all the currently existing _cursors_ in the current session.
 
```

Comment:
Under Scroll or No Scroll below, there is text "then allow scrolling is allowed" which may not be what you intend

Reply:
I _do_ intend this. It's a tricky call and impossible to discuss by essay exchange.